src/main/java/datastax/astra/migrate/OriginCountJobSession.java

```diff
@@ -77,9 +77,9 @@ public static OriginCountJobSession getInstance(CqlSession sourceSession, SparkC
 
     public void getData(BigInteger min, BigInteger max) {
         logger.info("ThreadID: {} Processing min: {} max: {}", Thread.currentThread().getId(), min, max);
-        int maxAttempts = maxRetries;
-        for (int retryCount = 1; retryCount <= maxAttempts; retryCount++) {
-
+        boolean done = false;
+        int maxAttempts = maxRetries + 1;
```

Comment:
#nitpick could we simplify it by removing an extra `maxAttempts` variable which is just used once in the for loop?

```
for (int attempts = 1; attempts <= (maxRetries + 1) && !done; attempts++) {
```

Same change for `DiffJobSession.java` and `CopyJobSession.java` as well.

Reply:
@msmygit It was added for code clarity but can be removed if you think its adding to confusion. Retries are now disabled by default & the logs report every attempt when it does not work